doc/Configuration.md

```diff
@@ -57,7 +57,7 @@ The example `configuration.json` options are explained:
     "Password": "",
     "Prefix": "umati/v2", // Topic prefix
     "ClientId": "MyCompany/ClientName", // ClientId part of topic structure
-    "Protocol": "tcp", // tcp: plain; tls: TLS secured; wss: WebSocket TLS secured
+    "Protocol": "wss", // tcp: plain; tls: TLS secured; wss: WebSocket TLS secured
```

Comment:
Port needs to be changed to 443 if wss here.

Reply:
yes